estimator/transport_co2/tests/test_transport_co2_model.py

```diff
@@ -0,0 +1,62 @@
+from transport_co2.model import Mode
+
+# Only distance
+def test_estimate_co2_bus():
+    co2_estimate = Mode["BUS"].estimate_co2(distance_in_km=10)
+
+    assert co2_estimate == 679.5275590551181
```

Comment:
Would you mind adding `approx()` to these tests as well?

Also, it's a little more direct to call `Mode.BUS`, no need for the square brackets here (though there's no harm in using them either).

Reply:
Oops, good catch.